dev/requires

```diff
@@ -1,18 +0,0 @@
-setuptools >= 39.0.1 ; python_version == '2.7' or python_version >= '3.3'
```

Comment:
This file should be restored, unless you can articulate how you've replaced all of the functionality.